bats/tests/helpers/utils.bash

```diff
@@ -209,24 +209,32 @@ calling_function() {
     echo "${FUNCNAME[2]}"
 }
 
-# Write a comment to the TAP stream
+# Write a comment to the TAP stream.
 # Set CALLER to print a calling function higher up in the call stack.
```

Comment:
This comment is now somewhat out-of-date because `CALLER` is only used when `RD_TRACE` is set.

I was at first confused that comments would get the same prefix as traces, but then realized that this is only done when `RD_TRACE` is true, so I guess that is fine.